Java/.idea/modules.xml

```diff
@@ -4,10 +4,13 @@
     <modules>
       <module fileurl="file://$PROJECT_DIR$/Core/Core.iml" filepath="$PROJECT_DIR$/Core/Core.iml" />
       <module fileurl="file://$PROJECT_DIR$/CoreCrypto/CoreCrypto.iml" filepath="$PROJECT_DIR$/CoreCrypto/CoreCrypto.iml" />
+      <module fileurl="file://$PROJECT_DIR$/CoreCrypto/CoreCrypto.iml" filepath="$PROJECT_DIR$/CoreCrypto/CoreCrypto.iml" />
```

Comment:
These duplicate entries aren't needed (Android Studio did this for some reason, happened to me too).